src/styles/blocks/general.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
don' use * selector for anything but box-sizing property
reset margins and padding for those elements you need
for instance

```scss
h1,
h2,
p,
//... {
margin: 0;
}
```

Reply:
Not fixed